eslint.config.mjs

```diff
@@ -0,0 +1,70 @@
+import js from '@eslint/js'
+import globals from 'globals'
+
+/** @type {import('eslint').Linter.FlatConfig[]} */
+export default [
+  // global ignores
+  {
+    'ignores': [
+      'bootstrap',
+      'node_modules/**'
+    ]
+  },
+  {
+    'languageOptions': {
+      'ecmaVersion': 2019,
+      'sourceType': 'script',
+      'globals': {
+        ...globals.node
+      }
+    },
+    'linterOptions': {
+      'reportUnusedDisableDirectives': 'error'
+    }
+  },
+  js.configs.recommended,
+  {
+    'files': [
+      '**/*.js'
+    ],
+    'rules': {
+      'comma-dangle': [
+        'error',
+        'never'
+      ],
+      'quotes': [
+        'error',
+        'single'
+      ],
+      'quote-props': [
+        'error',
+        'always'
+      ]
+    }
+  },
+  {
+    'files': [
+      '**/*.mjs'
+    ],
+    'languageOptions': {
+      'sourceType': 'module',
+      'globals': {
+        ...globals.nodeBuiltin
+      }
+    },
+    'rules': {
```

Comment:
I don't like this duplication, but unsure how to fix it. Maybe we could define a const with the common rules later?